tests/testthat/test.dbDeleteRowByKey.R

```diff
@@ -30,5 +30,12 @@ test_that("Exactly 50000 rows are deleted", {
 
 })
 
+test_that("The function will safly exit if given empty dt", {
+
+  # Expect there is no error
+  expect_error(dbDeleteRowByKey(db, "Letters", dt_letters[1 == 0]), NA)
```

Comment:
Got it from http://stackoverflow.com/questions/10826365/how-to-test-that-an-error-does-not-occur